From 773442f1bd95b5c2cbb42c678ddd82a17a0ae5fc Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 19 Dec 2023 19:38:09 +0100
Subject: [PATCH 0780/1141] dt-bindings: pci: stm32: add 'power-domains'
 property

STM32 PCIE-EP may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 PCIE-EP.

Fix dtbs_check warning:
Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ic2c30c435c7a7e06c905df8344b19ffdf3add0b1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347936
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 Documentation/devicetree/bindings/pci/st,stm32-pcie-ep.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/pci/st,stm32-pcie-ep.yaml b/Documentation/devicetree/bindings/pci/st,stm32-pcie-ep.yaml
index a9ae3a08fe70..03ee45f2960a 100644
--- a/Documentation/devicetree/bindings/pci/st,stm32-pcie-ep.yaml
+++ b/Documentation/devicetree/bindings/pci/st,stm32-pcie-ep.yaml
@@ -56,6 +56,9 @@ properties:
     description: GPIO controlled connection to PERST# signal
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
   feature-domains:
     maxItems: 1
 
-- 
2.39.2

